### PR TITLE
Fix #12745. Add connection profiles for Alibaba Cloud Object Storage …

### LINKED ISSUE #12745: Add connection profiles for Alibaba Cloud Object Storage Service (OSS)
As requested in iterate-ch/docs#118

### PR DESCRIPTION
**Pull request by dkocher**
…Service (OSS).

### CODE REVIEW

File: profiles/Alibaba (Beijing).cyberduckprofile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>alibaba-oss-cn-beijing</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Alibaba (Chengdu)</string>
```
Comment (AliveDevil) [inline profiles/Alibaba (Beijing).cyberduckprofile:27]:
```suggestion
        <string>Alibaba (Beijing)</string>
```

Comment (AliveDevil) [reply profiles/Alibaba (Beijing).cyberduckprofile:27]: `Alibaba China` or `Alibaba`?

File: profiles/Alibaba (Chengdu).cyberduckprofile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>alibaba-oss-cn-chengdu</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Alibaba China (Shanghai)</string>
```
Comment (AliveDevil) [inline profiles/Alibaba (Chengdu).cyberduckprofile:27]:
```suggestion
        <string>Alibaba China (Chengdu)</string>
```

File: profiles/Alibaba (Dubai).cyberduckprofile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>alibaba-oss-me-east-1</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Alibaba (Dubai)</string>
+        <key>Default Hostname</key>
+        <string>oss-me-east-1.aliyuncs.com</string>
```
Comment (AliveDevil) [inline profiles/Alibaba (Dubai).cyberduckprofile:29]: Inconsistent across profiles.
Sometimes `region.oss.aliyuncs.com` sometimes `oss-region.aliyuncs.com`.
Consolidate single variant.

File: profiles/Alibaba (Chengdu).cyberduckprofile (hunk 2 above)
Comment (AliveDevil) [reply profiles/Alibaba (Chengdu).cyberduckprofile:27]: `Alibaba China` or `Alibaba`? Inconsistent across profiles.

**Review by AliveDevil — CHANGES_REQUESTED** [review]
Consolidate usage of
* `Alibaba (City)` and `Alibaba China (City)`
* `oss-region.xyz` and `region.oss.xyz`

File: profiles/Alibaba (Dubai).cyberduckprofile (hunk 3 above)
Comment (dkocher) [reply profiles/Alibaba (Dubai).cyberduckprofile:29]: > 



> Inconsistent across profiles. Sometimes `region.oss.aliyuncs.com` sometimes `oss-region.aliyuncs.com`. Consolidate single variant.

This is intentional. When available we use an [endpoint that supports IPv6 to access OSS
](https://www.alibabacloud.com/help/en/doc-detail/31834.htm#section-iao-0i8-eel)

Comment (AliveDevil) [reply profiles/Alibaba (Dubai).cyberduckprofile:29]: 😠 Why can't those services make everything consistent.

**Review by AliveDevil — APPROVED** [review]

**Pull request merged**